requirements/base.txt

```diff
@@ -17,7 +17,7 @@ metafone==0.5
 le-utils==0.0.11
 kolibri_exercise_perseus_plugin==0.7.1
 jsonfield==2.0.1
-https://github.com/learningequality/morango/archive/ef260d263c92a4d3689e9ae742e25b57b52656e2.zip#egg=morango
+morango==0.1.1
```

Comment:
Flagging to update after morango fixes are made.

Reply:
looks like a lot of the PR template was wiped out: https://github.com/learningequality/kolibri/blob/develop/.github/PULL_REQUEST_TEMPLATE.md

In particular, note:

> Link to diff of internal dependency change is included